com.unity.netcode.gameobjects/Runtime/Transports/UTP/UnityTransport.cs

```diff
@@ -1117,11 +1129,13 @@ public override void Initialize(NetworkManager networkManager = null)
             // account for the overhead of its length when we store it in the send queue.
             var fragmentationCapacity = m_MaxPayloadSize + BatchedSendQueue.PerMessageOverhead;
 
-            m_NetworkSettings
-                .WithFragmentationStageParameters(payloadCapacity: fragmentationCapacity)
+            m_NetworkSettings.WithFragmentationStageParameters(payloadCapacity: fragmentationCapacity)
+#if !UTP_TRANSPORT_2_0_ABOVE
                 .WithBaselibNetworkInterfaceParameters(
                     receiveQueueCapacity: m_MaxPacketQueueSize,
-                    sendQueueCapacity: m_MaxPacketQueueSize);
+                    sendQueueCapacity: m_MaxPacketQueueSize)
```

Comment:
The receive/send queue capacities are now set through the `WithNetworkConfigParameters` call (in `CreateDriver`). It should thus be updated to something like this:

```csharp
m_NetworkSettings.WithNetworkConfigParameters(
    maxConnectAttempts: transport.m_MaxConnectAttempts,
    connectTimeoutMS: transport.m_ConnectTimeoutMS,
    disconnectTimeoutMS: transport.m_DisconnectTimeoutMS,
#if UTP_TRANSPORT_2_0_ABOVE
    sendQueueCapacity: m_MaxPacketQueueSize,
    receiveQueueCapacity: m_MaxPacketQueueSize,
#endif
    heartbeatTimeoutMS: transport.m_HeartbeatTimeoutMS);
```